CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Fixed
 - UI changes to 'Source Repo URL' field in catalog. [#814](https://github.com/rokwire/rokwire-building-blocks-api/issues/814)
+- Error in creating new contribution. [#858](https://github.com/rokwire/rokwire-building-blocks-api/issues/858)
```

Comment:
Please include the relevant issues in the CHANGELOG. Thanks.

Reply:
added